src/deploy/osp_deployer/director.py

```diff
@@ -408,6 +414,76 @@ def setup_templates(self):
         self.setup_environment()
         self.setup_sanity_ini()
 
+    def clamp_min_pgs(self, num_pgs):
+        pg_options = [8192, 4096, 2048, 1024, 512, 256, 128, 64, 32, 16, 8, 4,
+                      2, 1]
+        pg_max = pg_options[0]
```

Comment:
pg_max variable not used.  i believe this is a remnant of earlier experimentations

Reply:
removed.